ui/js/services/main.js

```diff
@@ -21,6 +21,12 @@ treeherder.factory('thUrl', [
             },
             getLogViewerUrl: function(job_id) {
                 return "logviewer.html#?job_id=" + job_id + "&repo=" + $rootScope.repoName;
+            },
+            getBugIdUrl:function(bug_id) {
```

Comment:
I think maybe the function name might be better as `getBugUrl` rather than `getBugIdUrl`?